### PR TITLE
add push to list of apps, update some text around ui, and add links

### PR DESCRIPTION
**Pull request by pwright**
addresses https://issues.redhat.com/browse/INTLY-4525
and some minor updates to other parts

### CODE REVIEW

File: modules/ROOT/pages/_partials/intro-con.adoc [hunk 1]
```diff
@@ -26,14 +28,16 @@ The {webapp} presents links to the applications available in a {PRODUCT_SHORT} c
 
 * *Red Hat Fuse Online*, a facility to obtain data from an application or service, operate on that data, and then send the data to another application or service without writing code.
 
-* *Red Hat Single Sign-On*, a single sign-on solution for web apps and RESTful web services.
+* *Push Notifications*, a service to push notifications to Android and iOS devices.
```
Comment (briandooley) [inline modules/ROOT/pages/_partials/intro-con.adoc:31]:
```suggestion
* *Push Notifications*, a service to push messages to Android and iOS devices.
```
Minor pickiness here but should we be saying 'notifications' twice in the same sentence?

File: modules/ROOT/pages/index.adoc [hunk 2]
```diff
@@ -8,8 +8,10 @@ For documentation on the latest release of RHMI, visit the link:https://access.r
   
 == Additional Resources
 
-* link:https://github.com/integr8ly/tutorial-web-app-walkthroughs[Integreatly Solution Patterns]
+* link:/data-sync/latest/[latest Data Sync documentation]
```
Comment (briandooley) [inline modules/ROOT/pages/index.adoc:11]: I assume that the hostname is automatically filled in here?  i.e that there is more to this link than /data-sync/latest/

**Review by briandooley — APPROVED** [review]
Couple of questions but lgtm

File: modules/ROOT/pages/_partials/intro-con.adoc (hunk 1 above)
Comment (pwright) [reply modules/ROOT/pages/_partials/intro-con.adoc:31]: don't think we can call them messages

File: modules/ROOT/pages/index.adoc (hunk 2 above)
Comment (pwright) [reply modules/ROOT/pages/index.adoc:11]: this is for https://integr8ly.netlify.com/ 
won't appear in any other contxt

**Pull request merged**